code/game/objects/structures/crates_lockers/closets/secure/medical.dm

```diff
@@ -1,6 +1,6 @@
 /obj/structure/closet/secure_closet/medical1
 	name = "Medicine Closet"
-	desc = "Filled with medical junk."
+	desc = "A closet containing medical supplies."
```

Comment:
This is just downgrading a snarky description (for something that needs no description) into a useless and bland description (for something that needs no description)